benchmark_v2/conftest.py

```diff
@@ -13,6 +13,7 @@
 
 default_dtype = ["int64", "uint64", "float64", "bool", "str", "bigint", "mixed"]
 default_encoding = ["ascii", "idna"]
+default_compression = ["none", "snappy", "gzip", "brotli", "zstd", "lz4"]
```

Comment:
I believe `"none"` should be `None` here based on how `aroukda.io.to_parquet` is configured.

Reply:
Updated